openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/TopicRepository.java

```diff
@@ -353,5 +355,15 @@ private void updateFieldDisplayName(Field origField, Field updatedField) throws
       String field = getSchemaField(original, origField, FIELD_DISPLAY_NAME);
       recordChange(field, origField.getDisplayName(), updatedField.getDisplayName());
     }
+
+    private void updateFieldDataTypeDisplay(Field origField, Field updatedField) throws JsonProcessingException {
+      if (operation.isPut() && !nullOrEmpty(origField.getDataTypeDisplay()) && updatedByBot()) {
+        // Revert the non-empty field description if being updated by a bot
+        updatedField.setDataTypeDisplay(origField.getDataTypeDisplay());
+        return;
+      }
+      String field = getSchemaField(original, origField, FIELD_DISPLAY_NAME);
```

Comment:
```suggestion
      String field = getSchemaField(original, origField, FIELD_DATA_DISPLAY_TYPE);
```

Define FIELD_DATA_DISPLAY_TYPE Field here